dags/common/france_travail/models.py

```diff
@@ -0,0 +1,55 @@
+from sqlalchemy import Column
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.types import DECIMAL, DateTime, Integer, String
+
+
+DB_SCHEMA = "france_travail"
+
+
+# NOTE: when upgrading to sqlalchemy 2.0 or higher, we'll need to use the class DeclarativeBase
+FranceTravailBase = declarative_base()
+
+
+class JobSeekerStats(FranceTravailBase):
+    """
+    Contains statistics on job-seeker by region and characteristic.
+    Imported from the France Travail API
+    https://francetravail.io/produits-partages/catalogue/marche-travail/documentation#/api-reference/operations/rechercherStatDemandeurs
+    """
+
+    __tablename__ = "job_seeker_stats"
+    __table_args__ = {"schema": DB_SCHEMA}
+
+    # Primary key fields.
+    codeCaract = Column(String, primary_key=True)
+    codeNomenclature = Column(String, primary_key=True)
+    codePeriode = Column(String, primary_key=True)
+    codeTerritoire = Column(String, primary_key=True)
+    codeTypeCaract = Column(String, primary_key=True)
+    codeTypeTerritoire = Column(String, primary_key=True)
+
+    codeActivite = Column(String)
+    codeTypeActivite = Column(String)
+    codeTypePeriode = Column(String)
+    datMaj = Column(DateTime)
+    libActivite = Column(String)
+    libCaract = Column(String)
+    libNomenclature = Column(String)
+    libPeriode = Column(String)
+    libTerritoire = Column(String)
+    nombre = Column(Integer)
+    pourcentage = Column(DECIMAL)
+
+    def __repr__(self) -> str:
+        return (
+            f"JobSeekerStats(codeCaract={self.codeCaract!r}, "
+            f"codeNomenclature={self.codeNomenclature!r}, "
+            f"codePeriode={self.codePeriode!r}, "
+            f"codeTerritoire={self.codePeriode!r}, "
+            f"codeTypeCaract={self.codePeriode!r}, "
+            f"codeTypeTerritoire={self.codePeriode!r})"
```

Comment:
PI, si le _repr_ ne permet de recréer l'objet à l'identique alors en général on utilise `<...>` pour le signifier : https://docs.python.org/reference/datamodel.html#object.__repr__
```suggestion
            f"<JobSeekerStats(codeCaract={self.codeCaract!r}, "
            f"codeNomenclature={self.codeNomenclature!r}, "
            f"codePeriode={self.codePeriode!r}, "
            f"codeTerritoire={self.codePeriode!r}, "
            f"codeTypeCaract={self.codePeriode!r}, "
            f"codeTypeTerritoire={self.codePeriode!r})>"
```